docs/release-notes/pytorch-tensorboard-plugin.rst

```diff
@@ -0,0 +1,10 @@
+:orphan:
+
+**Known Issue**
+
+-  PyTorch has `deprecated
+   <https://pytorch.org/tutorials/intermediate/tensorboard_profiler_tutorial.html#use-tensorboard-to-view-results-and-analyze-model-performance>`
+   their Profiler TensorBoard Plugin (``tb_plugin``), so some features may not be compatible with
+   PyTorch 2.0 and above. Our current default environment image comes with PyTorch 2.3. If users
+   experiencing issues with this plugin, we suggest using an image with PyTorch version earilier
```

Comment:
earilier -> earlier, if we get a chance to revise this before release

Reply:
Good catch! Revised: https://github.com/determined-ai/determined/pull/10183